src/app/api/activity-logs/services/activity-log.service.ts

```diff
@@ -82,9 +83,7 @@ export class ActivityLogService extends BaseService {
           companies,
         ),
         createdAt: activityLog.createdAt.toISOString(),
-        initiator: {
-          ...copilotUsers.find((iu) => iu.id === activityLog.userId),
-        },
+        initiator: initiator || null,
```

Comment:
After implementing previous fix:
```suggestion
        initiator,
```